src/harvest.cpp

```diff
@@ -44,7 +45,7 @@ const harvest_id &harvest_list::id() const
 
 std::string harvest_list::message() const
```

Comment:
Please also check where this function is used and remove redundant translation calls there.

Reply:
Looks like it's only called once in `butchery_drops_harvest` and there's no translation handling there.

Reply:
https://github.com/CleverRaven/Cataclysm-DDA/blob/71ee31c5e168b7e7e7f1e5223932fcc327daaa33/src/activity_handlers.cpp#L696

`_()` is the redundant translation call.